src/index.css

```diff
@@ -15,10 +19,150 @@ code {
 html{
   font-size: 125%;
   text-align: center;
+  background-image: url(../assets/redlightbackground\ \(2\).jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+h1,
+label,
+p,
+h2,
+h3{
+  color: white;
+}
+
+button{
+  background-color: white;
+  border: white 3px solid;
+  color: black;
+  transition: 0.5s;
+}
+
+button:hover{
+  background-color: lightcoral;
+  border: solid lightcoral 3px;
 }
-.anime {
+
+.link{
+  display: inline-block;
+  text-decoration: none;
+  color: white;
+  width: 5%;
+  transition: 0.5s;
+}
+
+.link:hover{
+  color: blue;
+}
+
+.container{
   display: flex;
   flex-direction: column;
-  width: 20%;
+  padding: 15px;
+  margin: 0 auto;
+}
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
```

Comment:
Make sure you add a `max-width: xxx px` and a `width: xx%` to get some marks in for the requirements! (Also cause people with big monitors can see your content properly too